artichoke-backend/src/extn/core/time/trampoline.rs

```diff
@@ -11,13 +12,17 @@ pub fn now(interp: &mut Artichoke) -> Result<Value, Error> {
     Ok(result)
 }
 
-pub fn at<T>(interp: &mut Artichoke, args: T) -> Result<Value, Error>
-where
-    T: IntoIterator<Item = Value>,
-{
-    let _ = interp;
-    let _ = args;
-    Err(NotImplementedError::new().into())
+pub fn at(interp: &mut Artichoke, seconds: Value, microseconds: Value) -> Result<Value, Error> {
```

Comment:
with the change in the mruby trampoline, this signature will take `Option<Value>` for `microseconds`:

```suggestion
pub fn at(interp: &mut Artichoke, seconds: Value, microseconds: Option<Value>) -> Result<Value, Error> {
```